lib/src/model/pdf_document_catalog.dart

```diff
@@ -24,6 +25,16 @@ class PDFDocumentCatalog {
     return PDFPages(pageRoot);
   }
 
+  /// Reads the document's outlines into a [List] of [PDFBookmark]s
+  Future<List<PDFBookmark>> getBookmarks() async {
```

Comment:
Since we try to make as little assumptions about structure as possible, we should return a nullable list here to be able to distinguish between no outlines and empty outlines